OnlineStore/store/src/main/java/StoreApp.java

```diff
@@ -1,26 +1,58 @@
-import java.lang.reflect.Field;
-import java.util.ArrayList;
+import product.ChainComparator;
+import product.ListProductComparators;
+import product.Product;
+import utility.DOMExecuter;
+
+
+import java.net.URL;
+import java.util.*;
+import java.util.concurrent.*;
 
 public class StoreApp {
     public static void main(String[] args) {
+        /**/
+
+        Store store = new Store(3000);
         RandomStorePopulator populator = new RandomStorePopulator();
```

Comment:
you ve created RandomStorePopulator several times, but i think there should be only one instance of it (and im tanking not about singleton), just create it onese, somewhere